src/layers/models/layersManager.ts

```diff
@@ -43,79 +42,64 @@ class LayersManager {
     return requestedLayer;
   }
 
-  public async addLayer(layerRequest: ILayerPostRequest): Promise<void> {
-    const editJson = (jsonDocument: IMapProxyJsonDocument): IMapProxyJsonDocument => {
-      this.addNewCache(jsonDocument, layerRequest);
-
-      return jsonDocument;
-    };
-
-    await this.configProvider.updateJson(editJson);
-  }
+  public async getCacheName(layerName: string, cacheType: string): Promise<ICacheName> {
+    const errorMsg = `Layer ${layerName} or fitting cache do not exist`
+    if (!this.isCacheTypeValid(cacheType)) {
+      const invalidCacheTypeMsg = `Invalid cache type: ${cacheType} has been provided , available values: "geopackage", "s3", "file", "redis"`
+      throw new Error(invalidCacheTypeMsg);
```

Comment:
Please add to OpenAPI scheme and return 400

Reply:
log warn with the 400 error and request params

Reply:
done